.gitignore

```diff
@@ -0,0 +1,130 @@
+###OSX###
```

Comment:
We use minimal ignore lists to make them easier to maintain.  Please remove everything here that isn't necessary for this project.